### PR TITLE
Silent parallelization

### PR DESCRIPTION
**Pull request by uukhas**
The order came from #269. Might be interesting for #373.

### CODE REVIEW

File: meta/Decays.m [hunk 1]
```diff
@@ -2238,7 +2238,8 @@ that returns a total (sumed over internal insertions) 1-loop amplitude for a giv
                  AbsoluteTiming@ParallelMap[
                     CreateTotalAmplitudeSpecialization[#, modelName]&,
                     Flatten[Last @@@ particleDecays, 1],
-                    DistributedContexts -> All, Method -> "FinestGrained"
+                    DistributedContexts -> None,
```
Comment (wkotlarski) [inline meta/Decays.m:2241]: With `DistributedContexts -> None` the calculation is not parallelized.

Comment (uukhas) [reply meta/Decays.m:2241]: Yes, now I see it. Thank you! Let me look at more carefully.

**Comment by uukhas** [comment]
It seems that I was too optimistic and naive to think that the problem of shadowing can be solved by the permutation of the names for packages.

Thank you, @wkotlarski, that you took a look at my suggestion, but it seems for me now, that the problem is much deeper, so it would be better to have some **general solution** in some Issue before making any PR.

**Comment by Expander** [comment]
Just a side comment: This reordering problem is exactly the type of problems that we have all over the place, see also issue #374 . @uukhas I will experiment with your suggestion that you proposed in issue #374, thanks! Generally, it would be best to have a more general solution to such shadowing problems (and similar issues).

**Pull request closed without merging**